gen/dcos-config.yaml

```diff
@@ -1765,7 +1765,23 @@ package:
               },
               {
                   "Location": "{{ mesos_master_log_file }}",
-                  "Role": ["master"]
+                  "Role": ["master"],
+                  "Optional": true
+              },
+              {
+                  "Location": "{{ mesos_master_log_file }}.1",
```

Comment:
Should this be `*.1.gz`, or are we expecting it not to be gzipped?

Reply:
I took this list from this comment https://github.com/dcos/dcos/pull/3589#discussion_r223855676
@kaysoky Can you confirm it's OK?

Reply:
Yes, we use the `delaycompress` option of logrotate, which means the first rotated file will not be compressed.

https://github.com/dcos/dcos/blob/226f490f6769986a832c8fc8613c71c9cd7f45ac/gen/dcos-config.yaml#L1256

Reply:
Listing out these files probably is not the ideal way to grab the logs.  The number of logs is configurable, and by default, goes up to `*.136.gz`.

https://github.com/dcos/dcos/blob/226f490f6769986a832c8fc8613c71c9cd7f45ac/gen/calc.py#L213-L218

Reply:
Agree but we won't include them all in diagnostics bundle. How much of them do you need?

Reply:
If it is impractical to grab everything, then the existing amount (4 files, 0.75 - 1 GB total) is a reasonable middle ground, assuming diagnostic bundles are grabbed shortly after spotting a problem/reporting.